bedrock/mozorg/templatetags/misc.py

```diff
@@ -684,7 +684,7 @@ def slugify(text):
 
 @library.filter
 def bleach_tags(text):
-    return bleach.clean(text, tags=[], strip=True).replace("&amp;", "&")
+    return bleach.clean(text, tags={}, strip=True).replace("&amp;", "&")
```

Comment:
Looks like empty `{}` returns a type `dict` rather than a type `set`.

Reply:
Yeah good call - will amend now. Tests pass at least, but better to be explicit